Keas.Mvc/package.json

```diff
@@ -5,9 +5,11 @@
   "scripts": {
     "debug": "ASPNETCORE_ENVIRONMENT=Development dotnet watch run",
     "webpack": "node node_modules/webpack/bin/webpack.js --env.dev --watch",
+    "devpack": "node node_modules/webpack-dev-server/bin/webpack-dev-server.js --config webpack.config.js",
```

Comment:
we can do this in a separate PR, but for other apps i've included the `--hot` param on here in order to get hot reloading working.  it probably requires a bit of other config too but might explain part of why hot reloading isn't working yet

Reply:
As far as I'm aware, it's equivalent to the hot: true setting I put inside the webpack.config.

Reply:
I tried it both ways, and it didn't make any difference.